stdlib/public/core/FloatingPoint.swift

```diff
@@ -328,15 +328,20 @@ public protocol FloatingPoint: SignedNumeric, Strideable, Hashable
   /// `infinity` is greater than this value.
   static var greatestFiniteMagnitude: Self { get }
 
-  /// The mathematical constant pi.
+  /// The [mathematical constant π][wiki], approximately equal to 3.14159.
+  /// 
+  /// The π constant is equivalent to 180°, or half a revolution of a circle,
```

Comment:
Or even shorter, "When measuring an angle in radians, π is equivalent to 180°."

Reply:
Hmm, I like the conciseness, but the structure is less nice because it moves the assertion to the end of the sentence (whereas the current version starts with the assertion and adds detail for clarification). But I think I can tidy up a bit more.

Reply:
Updated.